src/components/UI/navbar/index.js

```diff
@@ -27,7 +27,7 @@ const GenericClosePopUp = (ref, callback) => {
     };
   });
 };
-
+let authUrl = '';
```

Comment:
change this into a useRef

Reply:
Why you want to use useRef, 
for same ref object on renders or what else?